homeassistant/components/deconz/services.py

```diff
@@ -67,7 +75,14 @@ async def async_call_deconz_service(service_call):
         DOMAIN,
         SERVICE_DEVICE_REFRESH,
         async_call_deconz_service,
-        schema=SERVICE_DEVICE_REFRESH_SCHEMA,
+        schema=SELECT_GATEWAY_SCHEMA,
+    )
+
+    hass.services.async_register(
+        DOMAIN,
+        SERVICE_REMOVE_ORPHANED_ENTRIES,
```

Comment:
Why make this a service? Can't we do this on startup or at an interval (every night etc)

Reply:
Why not on restart: lights is not loaded if it's a full system restart so would remove too much

Could possibly be called at night but the best time is some time after a restart which needs to be handled manually

Reply:
Wouldn't it be save to do automatically 2h after boot? It seems weird to put the burden of this on the user.

Reply:
Hey! Why the service is because you can never know for sure when a system has restored properly after a system restart. This is due to Deconz loads lights too slow meaning that the integration usually doesn't get the lights loaded on startup but requires either a call to the other service device refresh or to restart core or the integration it self. So that's why the user needs to decide when to do it